docs/modules/operation/pages/deep-dive/performance-data-collection/collectd/collection-packages.adoc

```diff
@@ -23,16 +23,16 @@ At a minimum, collector package attributes include a package name and a filter t
 <package name="cassandra-via-jmx" remote="false">
       <filter>IPADDR != '0.0.0.0'</filter>
 ----
+
 Note that `remote="false"` means that the services in this package are tested only from the OpenNMS core system itself and not from a different remote location.
 
-Each package must have a filter tag that performs the initial test to see if an interface should be included in a package.
+Each package must have a `filter` tag that performs the initial test to see if an interface should be included in a package.
```

Comment:
Let's change to: "is eligible to be included in a package", since passing the filter does not guarantee inclusion.

Reply:
```suggestion
Each package must have a `filter` tag that performs the initial test to see if an interface is eligible to be included in a package.
```